protos/dcs/mission/v0/mission.proto

```diff
@@ -443,6 +443,14 @@ message StreamEventsResponse {
     google.protobuf.Struct details = 2;
   }
 
+  /**
+   * Fired every second containing the FPS since the previous event.
+   */
+  message FpsEvent {
```

Comment:
Wdyt, should it be called `ServerFpsEvent`?

Reply:
I think `ServerFpsEvent` is the better name or maybe a more accurate `SimulationFPSEvent`  / `ServerSimulationFPSEvent` (It is a longer name but more accurate and I don't mind long names if there is a reason. However I am fine with any of the names to be honest).

Reply:
I would concur with `SimulationFpsEvent` -- I have kind of started work on a single-player mechanic (so not necessarily it's a server)

Reply:
I like `SimulationFpsEvent` 👍 - I'll change it in the next update of the PR